src/WorkdayWebhookReceiverService.Web/Controllers/WorkdaywebhookreceiverController.cs

```diff
@@ -13,5 +15,12 @@ public WorkdayWebhookReceiverServiceController(Core.Services.WorkdayWebhookRecei
             _workdayWebhookReceiverServiceService = workdayWebhookReceiverServiceService;
         }
         
+        [HttpPost ("Worker")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status503ServiceUnavailable)]
+        public async Task<IActionResult> WorkerUpdate([FromBody] Worker worker)
```

Comment:
```suggestion
        public async Task<IActionResult> UpdateWorker([FromBody] Worker worker)
```

I reeaeeeaaalllly like the verb in front. I'll buy you a beer. :D